sdk/keyvault/azure-security-test-keyvault-jca/pom.xml

```diff
@@ -21,7 +21,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-jca</artifactId>
-      <version>1.0.0-beta.8</version> <!-- {x-version-update;com.azure:azure-security-keyvault-jca;current} -->
+      <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-security-keyvault-jca;current} -->
```

Comment:
Maybe we should keep it to `1.0.0-beta.1`

Reply:
can we?